src/twisted/topfiles/9100.bugfix

```diff
@@ -0,0 +1 @@
+twisted.mail.pop3 now accepts passwords that contain spaces.
```

Comment:
Very nice.  I'd say `twisted.mail.pop3.POP3 ...` or `The POP3 server implemented by twisted.mail.pop3 ...` though to avoid client/server confusion